terraform/dev-app.tfvars

```diff
@@ -0,0 +1,9 @@
+﻿Instance_type          = "t2.medium"
```

Comment:
Are we likely to actually need a medium here? Have we looked into whether we could get away with a small on dev and pre-prod?

Reply:
We should be able to get away with a small and I've made the changes.